[2022-02-02 21:49:51,145] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_Zone.halo scheduled__2022-01-01T00:00:00+00:00 [queued]>
[2022-02-02 21:49:51,184] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_Zone.halo scheduled__2022-01-01T00:00:00+00:00 [queued]>
[2022-02-02 21:49:51,185] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:49:51,185] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-02 21:49:51,186] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:49:51,227] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): halo> on 2022-01-01 00:00:00+00:00
[2022-02-02 21:49:51,236] {standard_task_runner.py:52} INFO - Started process 2539 to run task
[2022-02-02 21:49:51,242] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question4_Zone', 'halo', 'scheduled__2022-01-01T00:00:00+00:00', '--job-id', '384', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions4.py', '--cfg-path', '/tmp/tmpe5w3_d4e', '--error-file', '/tmp/tmpzvfy18cq']
[2022-02-02 21:49:51,244] {standard_task_runner.py:77} INFO - Job 384: Subtask halo
[2022-02-02 21:49:51,411] {logging_mixin.py:109} INFO - Running <TaskInstance: Question4_Zone.halo scheduled__2022-01-01T00:00:00+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:49:51,599] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question4_Zone
AIRFLOW_CTX_TASK_ID=halo
AIRFLOW_CTX_EXECUTION_DATE=2022-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-01T00:00:00+00:00
[2022-02-02 21:49:51,601] {logging_mixin.py:109} INFO - hello world!!
[2022-02-02 21:49:51,601] {python.py:175} INFO - Done. Returned value was: None
[2022-02-02 21:49:51,639] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question4_Zone, task_id=halo, execution_date=20220101T000000, start_date=20220202T214951, end_date=20220202T214951
[2022-02-02 21:49:51,722] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:49:51,816] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
